test/tests/functional/pbs_svr_dyn_res.py

```diff
@@ -214,9 +213,9 @@ def test_res_whitespace(self):
 
         fpath_out = os.path.join(os.sep, "tmp", "PtlPbs_got_foo")
 
-        fn_in = self.du.create_temp_file(prefix="PtlPbs_get_foo",
-                                         body=script_body, asuser=ROOT_USER)
-        self.du.chmod(path=fn_in, mode=0755, sudo=True)
+        fn_in = self.du.create_dyn_res_script(script_body,
+                                              prefix="PtlPbs_get_foo")
+        self.filenames.append(fn_in)
```

Comment:
just curious, did previously the temp files just stick around after running the test?

Reply:
I was just making sure that teardown of each test clear files created by them. This is because PTL's cleanup is dependent on the file name prefixed with PtlPbs* and there are bunch of tests that creates files with different prefixes. So I'm going to do cleanup for those files, might as well do it for others.